bedrock/mozorg/middleware.py

```diff
@@ -45,3 +46,17 @@ def process_view(self, request, view, view_args, view_kwargs):
         else:
             f = super(MozorgRequestTimingMiddleware, self)
             f.process_view(request, view, view_args, view_kwargs)
+
+
+class CrossOriginResourceSharingMiddleware(object):
```

Comment:
I thought the preflighting was done by the browser on a CORS request, and so would always happen. I could easily be wrong on this, but it is clear that a preflight check would fail with this. Would that make it possibly fail in some browsers and not others?